feature-libs/cart/main/components/cart/index.ts

```diff
@@ -1,6 +1,5 @@
-export * from './add-to-cart/add-to-cart.component';
-export * from './add-to-cart/add-to-cart.module';
-export * from './add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component';
+export * from './added-to-cart-dialog/added-to-cart-dialog.component';
+export * from './added-to-cart-dialog/added-to-cart-dialog.module';
```

Comment:
just this `export * from './added-to-cart-dialog/index';`.

Reply:
Yes, the index is better.